### PR TITLE
feat(index): base Pingdom check detection on user agent

### LINKED ISSUE #17: Simplify Pingdom request detection
**Is your feature request related to a problem? Please describe.**
The current heuristics to detect a Pingdom status check request are to complicated and restrictive. Pingdom request detection is e.g. based on the absence of query parameters. It's therefore not possible to use real world action invocations for status checks.

**Describe the solution you'd like**
Simplify the heuristics by just using the `User-Agent` header to determine whether it is a Pingdom status check request or not.

### PR DESCRIPTION
**Pull request by stefan-guggisberg**
BREAKING CHANGE: The behaviour of the wrapper changed. Pingdom status response generation depends on
user agent instead of Http method and absence of parameters.

fix #17

Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

**Comment by stefan-guggisberg** [comment]
@tripodsan I agree regarding guessing whether it's a Pingdom request or not based on method and absence of parameters. This PR replaces the intransparent guessing with a deterministic test based on the user agent. 

This PR addresses an issue in a Pingdom specific repo (`helix-pingdom-status`). For generic (non-Pingdom specific) status reporting we should create a new issue & PR.

**Comment by stefan-guggisberg** [comment]
> ```
> GET /_status_check/pingdom.xml
>
>
> HTTP/1.1 200 OK
>
> <xml>
>  <status>Service Unavailable</status>
>  <error>runtime is down</error>
></xml>
>```

This would be useless for Pingdom Http checks as it wouldn't trigger an incident (status != 200).

**Comment by stefan-guggisberg** [comment]
I suggest we fix the current Pingdom checks since they don't provide the information needed to conduct an RCA (e.g. the Activation Id).

Introducing generic status reports/health checks or switching to a different monitoring service provider should be subject of a separate discussion/issue.

**Comment by tripodsan** [comment]
but I really would not rely on user agent. you can't ensure that they use this user-agent for all their checks, and it makes it very hard to check/test with a browser, or cumbersome to test with curl. 

I would rather use an url as suggested, eg:

`/_status_check/pingdom.xml`

and then we change all our check-urls in pingdom. 

---

as for using a different return behaviour, this can be done in a different issue. I agree.

**Comment by stefan-guggisberg** [comment]
> I would rather use an url as suggested, eg:
/_status_check/pingdom.xml
and then we change all our check-urls in pingdom.

Ok, agreed.

File: README.md [hunk 1]
```diff
@@ -36,10 +36,10 @@ Finally, you know that there are [HTTP Custom Checks](https://help.pingdom.com/h
 The service is installed and available for Adobe I/O Runtime at `https://adobeioruntime.net/api/v1/web/helix/helix-services/pingdom-status@v1`. 
 
 ```bash
-curl https://adobeioruntime.net/api/v1/web/helix/helix-services/pingdom-status@v1
+curl https://adobeioruntime.net/api/v1/web/helix/helix-services/pingdom-status@v1/_status_check/pingdom.xml
 ```
 
-If you want to monitor the availability of Adobe I/O Runtime, just add a new [HTTP Custom Check](https://help.pingdom.com/hc/en-us/articles/115000431709-HTTP-Custom-Check) in Pingdom, using the `https://` protocol, and `adobeioruntime.net/api/v1/web/helix/helix-services/pingdom-status@v1` as the URL.
+If you want to monitor the availability of Adobe I/O Runtime, just add a new [HTTP Custom Check](https://help.pingdom.com/hc/en-us/articles/115000431709-HTTP-Custom-Check) in Pingdom, using the `https://` protocol, and `adobeioruntime.net/api/v1/web/helix/helix-services/pingdom-status@v1/_status_check/pingdom.xmls` as the URL.
```
Comment (tripodsan) [inline README.md]: typo: `xmls`

**Pull request merged**